packages/i18n/package.json

```diff
@@ -20,8 +20,8 @@
   },
   "main": "./dist/i18n-index.cjs.js",
   "module": "./dist/i18n-index.es.js",
-  "typings": "./dist/typings/index.d.ts",
-  "types": "./dist/typings/index.d.ts",
+  "typings": "./dist/typings/src/index.d.ts",
+  "types": "./dist/typings/src/index.d.ts",
```

Comment:
Somehow for the i18n and the l10n package the type declarations are put into a `src` folder

<img width="193" alt="image" src="https://user-images.githubusercontent.com/1110551/71619799-784c3280-2bc6-11ea-960d-cbe79818b2fd.png">

This is probably because of the `data` folder (I haven't been able to exclude it)